docs/source/changelog.rst

```diff
@@ -12,6 +12,55 @@ For more detailed information, see
      Use ``pip install notebook --upgrade`` or ``conda upgrade notebook`` to
      upgrade to the latest release.
 
+.. _release-5.0.0:
+
+Notebook version 5.0
+====================
+
+This is the first major release of the Notebook Application and Server after
+the version 4.0 created by the "Big Spilt" of IPython into IPython and Jupyter.
+
+A reminder that most of the development cycles are now on JupyterLab and that
```

Comment:
"We encourage users to start trying JupyterLab in preparation for a future transition." ?